Changelog.md

```diff
@@ -4,6 +4,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0] - 2020-07-20
+
```

Comment:
Let's add some kind of note here, even if it's simply to say that this is an official release with no notable changes from 0.1.6